CMakeLists.txt

```diff
@@ -79,15 +83,11 @@ ament_export_dependencies(
   Eigen3
   eigen3_cmake_module  # export Eigen3 headers
   Boost
-  ASSIMP
```

Comment:
> * Remove ASSIMP export dependency

is a bad commit message. I still have no idea why this is necessary, especially because you actually *add* OCTOMAP at the same time.

Reply:
ASSIMP doesn't need to be exported and also isn't for ROS1. Also, the cmake policy flag (https://github.com/ros-planning/geometric_shapes/pull/153/files#diff-af3b638bc2a3e6c650974192a53c7291R18) is only scoped to this file and can't be passed on so that all downstream dependencies would have to include this fix as well in order to suppress the related warning.